pkg/apiserver/rest/dualwriter.go

```diff
@@ -260,3 +295,46 @@ func getName(o runtime.Object) string {
 	}
 	return accessor.GetName()
 }
+
+const dataSyncerInterval = 60 * time.Minute
+
+// StartPeriodicDataSyncer starts a background job that will execute the DataSyncer every 60 minutes
+func StartPeriodicDataSyncer(ctx context.Context, mode DualWriterMode, legacy LegacyStorage, storage Storage,
+	kind string, reg prometheus.Registerer, serverLockService ServerLockService, requestInfo *request.RequestInfo) {
+	klog.Info("Starting periodic data syncer for mode mode: ", mode)
+
+	// run in background
+	go func() {
+		// run it immediately
+		_, _ = runDataSyncer(ctx, mode, legacy, storage, kind, reg, serverLockService, requestInfo)
+
+		ticker := time.NewTicker(dataSyncerInterval)
+		for {
+			select {
+			case <-ticker.C:
+				_, _ = runDataSyncer(ctx, mode, legacy, storage, kind, reg, serverLockService, requestInfo)
```

Comment:
we don't seem to be doing anything with these return values, when we `return everythingSynced, nil` from `mode2DataSyncer`. Should we instead not return anything? Or would you have other idea?

Reply:
it may be a good idea here to check the return value and log the result so we have some visibility into what the syncer is doing.

Reply:
> it may be a good idea here to check the return value and log the result so we have some visibility into what the syncer is doing.

Agree with that. I will adjust the code to log the returned value.